src/network/defaultlocalconnector.cpp

```diff
@@ -106,7 +106,7 @@ void DefaultLocalConnector::clearOutdatedAttributesOnTimerTick()
     if (!outdatedServerNames.empty())
     {
         bool isAnyConnectedAttributesOutdated = false;
-        for(const auto & serverName : outdatedServerNames)
+        for(const auto &serverName : outdatedServerNames)
```

Comment:
for (const auto &serverName : outdatedServerNames)

Reply:
fixed